genmc-sys/src/lib.rs

```diff
@@ -20,11 +38,153 @@ mod ffi {
         pub do_symmetry_reduction: bool,
         // FIXME(GenMC): Add remaining parameters.
     }
+
+    #[derive(Debug)]
+    enum ActionKind {
+        /// Any Mir terminator that's atomic and has load semantics.
+        Load,
```

Comment:
The comment should make it clear that "load" is "maybe load" and "non-load" is "*definitely* not a load".